plugins/jobs/src/js/JobDelete.js

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+import { componentFromStream, graphqlObservable } from "data-service";
+import { Subject } from "rxjs/Subject";
+import "rxjs/add/operator/map";
+import "rxjs/add/operator/switchMap";
+import "rxjs/add/operator/do";
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/startWith";
+import "rxjs/add/operator/catch";
+
+import gql from "graphql-tag";
+
+import JobDeleteModal from "./components/JobDeleteModal";
+import defaultSchema from "./data/JobModel";
+
+const deleteJob = gql`
```

Comment:
🎨 might be only me, but I find the naming confusing. I went ahead and looked up the definition of `graphqlObservable` because I thought a function would be passed to it